docs/DOCUMENTATION.md

```diff
@@ -1,3 +1,10 @@
 #### Formatting code
  Prettier is installed in this repo and will auto-format files on git commits using the Husky tool.
-Prettier automatically analyzes your code and rewrites it according to a set of predefined rules. It takes care of indentation, line breaks, spacing, and other formatting aspects, ensuring that the resulting code is easy to read and understand.
\ No newline at end of file
+Prettier automatically analyzes your code and rewrites it according to a set of predefined rules. It takes care of indentation, line breaks, spacing, and other formatting aspects, ensuring that the resulting code is easy to read and understand.
+
+#### prop-types for documenting component props
+Runtime type checking for React props and similar objects.
+You can use prop-types to document the intended types of properties passed to components. React will check props passed to your components against those definitions, and warn in development if they don’t match.
+
+#### Mirage library for mocking backend
```

Comment:
Might be good to add a sentence to tell the other developers where/what file to use to add routes to Mirage.

Reply:
I just add the sentence.